gemrb/core/System/VFS.cpp

```diff
@@ -247,18 +247,17 @@ void PathAppend(path_t& target, const path_t& name)
 		return;
 	}
 
-	if (!target.empty() && target.back() != PathDelimiter && name.front() != PathDelimiter) {
-		target.push_back(PathDelimiter);
+	auto scope = StringView{name.c_str(), name.length()};
+	// TotL has '\data\zcMHar.bif' in the key file
+	if (name[0] == '\\') {
```

Comment:
I don't think this should be part of the VFS. is there more than one place that has this issue?

Reply:
I agree about the misplacement. But I don't know any else r/n. This is mostly by-work since I wanted to address the first finding from the prework.